From e722b1e5f214ec1d007f9b856b97293df16ea4fb Mon Sep 17 00:00:00 2001
From: Kai Uwe Broulik <kde@privat.broulik.de>
Date: Fri, 26 Feb 2016 14:22:38 +0100
Subject: [PATCH 5/5] Fix typo

---
 components/sessionsprivate/sessionsmodel.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/sessionsprivate/sessionsmodel.cpp b/components/sessionsprivate/sessionsmodel.cpp
index 600e682..1efbe52 100644
--- a/components/sessionsprivate/sessionsmodel.cpp
+++ b/components/sessionsprivate/sessionsmodel.cpp
@@ -48,7 +48,7 @@ SessionsModel::SessionsModel(QObject *parent)
                 emit switchedUser(m_pendingVt);
             } else if (m_pendingReserve) {
                 m_displayManager.startReserve();
-                emit startNewSession();
+                emit startedNewSession();
             }
 
             m_pendingVt = 0;
-- 
2.5.0

