diff -up plasma-workspace-5.19.5/lookandfeel/contents/lockscreen/MainBlock.qml.0305 plasma-workspace-5.19.5/lookandfeel/contents/lockscreen/MainBlock.qml
--- plasma-workspace-5.19.5/lookandfeel/contents/lockscreen/MainBlock.qml.0305	2020-10-07 13:37:21.605019821 -0500
+++ plasma-workspace-5.19.5/lookandfeel/contents/lockscreen/MainBlock.qml	2020-10-07 13:40:07.792066613 -0500
@@ -97,7 +97,8 @@ SessionManagementScreen {
         PlasmaComponents.Button {
             id: loginButton
             Accessible.name: i18nd("plasma_lookandfeel_org.kde.lookandfeel", "Unlock")
-            implicitHeight: passwordBox.height - units.smallSpacing * 0.5 // otherwise it comes out taller than the password field
+            Layout.preferredHeight: passwordBox.implicitHeight
+            Layout.preferredWidth: loginButton.Layout.preferredHeight
 
             PlasmaCore.IconItem { // no iconSource because if you take away half a unit (implicitHeight), "go-next" gets cut off
                     anchors.fill: parent
diff -up plasma-workspace-5.19.5/sddm-theme/Login.qml.0305 plasma-workspace-5.19.5/sddm-theme/Login.qml
--- plasma-workspace-5.19.5/sddm-theme/Login.qml.0305	2020-10-07 13:37:21.605019821 -0500
+++ plasma-workspace-5.19.5/sddm-theme/Login.qml	2020-10-07 13:39:04.185665965 -0500
@@ -112,8 +112,8 @@ SessionManagementScreen {
         PlasmaComponents.Button {
             id: loginButton
             Accessible.name: i18nd("plasma_lookandfeel_org.kde.lookandfeel", "Log In")
-            implicitHeight: passwordBox.height - units.smallSpacing * 0.5 // otherwise it comes out taller than the password field
-            Layout.rightMargin: 1 // prevents it from extending beyond the username field
+            Layout.preferredHeight: passwordBox.implicitHeight
+            Layout.preferredWidth: loginButton.Layout.preferredHeight
 
             PlasmaCore.IconItem { // no iconSource because if you take away half a unit (implicitHeight), "go-next" gets cut off
                 anchors.fill: parent
