diff -up plasma-workspace-5.7.4/applets/systemtray/package/contents/ui/main.qml.orig plasma-workspace-5.7.4/applets/systemtray/package/contents/ui/main.qml
--- plasma-workspace-5.7.4/applets/systemtray/package/contents/ui/main.qml.orig	2016-08-24 06:15:17.781268751 -0500
+++ plasma-workspace-5.7.4/applets/systemtray/package/contents/ui/main.qml	2016-08-24 06:17:35.647239516 -0500
@@ -31,7 +31,7 @@ MouseArea {
     Layout.minimumHeight: vertical ? tasksRow.implicitHeight + (expander.visible ? expander.implicitHeight : 0) + units.smallSpacing : units.smallSpacing
 
     property bool vertical: plasmoid.formFactor == PlasmaCore.Types.Vertical
-    property int itemSize: units.roundToIconSize(Math.min(Math.min(width, height), units.iconSizes.medium))
+    property int itemSize: units.roundToIconSize(Math.min(Math.min(width, height), units.iconSizes.smallMedium))
     property int hiddenItemSize: units.iconSizes.smallMedium
     property alias expanded: dialog.visible
     property Item activeApplet
