diff --git a/shell/packageplugins/lookandfeel/lookandfeel.cpp b/shell/packageplugins/lookandfeel/lookandfeel.cpp
index d50b3e96f..556877af6 100644
--- a/shell/packageplugins/lookandfeel/lookandfeel.cpp
+++ b/shell/packageplugins/lookandfeel/lookandfeel.cpp
@@ -10,7 +10,7 @@
 #include <KLocalizedString>
 #include <KPackage/PackageLoader>
 
-#define DEFAULT_LOOKANDFEEL "org.kde.breeze.desktop"
+#define DEFAULT_LOOKANDFEEL "@DEFAULT_LOOKANDFEEL@"
 
 void LookAndFeelPackage::initPackage(KPackage::Package *package)
 {
diff --git a/startkde/startplasma.cpp b/startkde/startplasma.cpp
index bd848dda0..43978a712 100644
--- a/startkde/startplasma.cpp
+++ b/startkde/startplasma.cpp
@@ -365,7 +365,7 @@ void setupPlasmaEnvironment()
     qputenv("XDG_CONFIG_DIRS", extraConfigDir + ":" + currentConfigDirs);
 
     const KConfig globals;
-    const QString currentLnf = KConfigGroup(&globals, QStringLiteral("KDE")).readEntry("LookAndFeelPackage", QStringLiteral("org.kde.breeze.desktop"));
+    const QString currentLnf = KConfigGroup(&globals, QStringLiteral("KDE")).readEntry("LookAndFeelPackage", QStringLiteral("@DEFAULT_LOOKANDFEEL@"));
     QFile activeLnf(QString::fromUtf8(extraConfigDir + "/package"));
     activeLnf.open(QIODevice::ReadOnly);
     if (activeLnf.readLine() != currentLnf.toUtf8()) {
