diff --git a/shell/shellcorona.cpp b/shell/shellcorona.cpp
index 4aff26e..864e78a 100644
--- a/shell/shellcorona.cpp
+++ b/shell/shellcorona.cpp
@@ -306,6 +306,7 @@ void ShellCorona::load()
     checkActivities();
     if (containments().isEmpty()) {
         loadDefaultLayout();
+        processUpdateScripts();
     } else {
         processUpdateScripts();
         foreach(Plasma::Containment *containment, containments()) {
