diff -up plasma-workspace-5.11.4/startkde/startkde.cmake.startkde plasma-workspace-5.11.4/startkde/startkde.cmake
--- plasma-workspace-5.11.4/startkde/startkde.cmake.startkde	2017-11-28 06:44:10.000000000 -0600
+++ plasma-workspace-5.11.4/startkde/startkde.cmake	2017-12-19 10:31:04.000454158 -0600
@@ -158,48 +158,6 @@ for prefix in `echo $scriptpath`; do
   done
 done
 
-# Activate the kde font directories.
-#
-# There are 4 directories that may be used for supplying fonts for KDE.
-#
-# There are two system directories. These belong to the administrator.
-# There are two user directories, where the user may add her own fonts.
-#
-# The 'override' versions are for fonts that should come first in the list,
-# i.e. if you have a font in your 'override' directory, it will be used in
-# preference to any other.
-#
-# The preference order looks like this:
-# user override, system override, X, user, system
-#
-# Where X is the original font database that was set up before this script
-# runs.
-
-usr_odir=$HOME/.fonts/kde-override
-usr_fdir=$HOME/.fonts
-
-if test -n "$KDEDIRS"; then
-  kdedirs_first=`echo "$KDEDIRS"|sed -e 's/:.*//'`
-  sys_odir=$kdedirs_first/share/fonts/override
-  sys_fdir=$kdedirs_first/share/fonts
-else
-  sys_odir=$KDEDIR/share/fonts/override
-  sys_fdir=$KDEDIR/share/fonts
-fi
-
-# We run mkfontdir on the user's font dirs (if we have permission) to pick
-# up any new fonts they may have installed. If mkfontdir fails, we still
-# add the user's dirs to the font path, as they might simply have been made
-# read-only by the administrator, for whatever reason.
-
-test -d "$sys_odir" && xset +fp "$sys_odir"
-test -d "$usr_odir" && (mkfontdir "$usr_odir" ; xset +fp "$usr_odir")
-test -d "$usr_fdir" && (mkfontdir "$usr_fdir" ; xset fp+ "$usr_fdir")
-test -d "$sys_fdir" && xset fp+ "$sys_fdir"
-
-# Ask X11 to rebuild its font list.
-xset fp rehash
-
 # Set a left cursor instead of the standard X11 "X" cursor, since I've heard
 # from some users that they're confused and don't know what to do. This is
 # especially necessary on slow machines, where starting KDE takes one or two
@@ -276,22 +234,21 @@ if test $? -ne 0; then
   # Startup error
   echo 'startkde: Could not sync environment to dbus.'  1>&2
   test -n "$ksplash_pid" && kill "$ksplash_pid" 2>/dev/null
-  xmessage -geometry 500x100 "Could not sync environment to dbus."
+  message "Could not sync environment to dbus."
   exit 1
 fi
 
-# We set LD_BIND_NOW to increase the efficiency of kdeinit.
-# kdeinit unsets this variable before loading applications.
-LD_BIND_NOW=true @CMAKE_INSTALL_FULL_LIBEXECDIR_KF5@/start_kdeinit_wrapper --kded +kcminit_startup
+@CMAKE_INSTALL_FULL_LIBEXECDIR_KF5@/start_kdeinit_wrapper --kded +kcminit_startup
 if test $? -ne 0; then
   # Startup error
   echo 'startkde: Could not start kdeinit5. Check your installation.'  1>&2
   test -n "$ksplash_pid" && kill "$ksplash_pid" 2>/dev/null
-  xmessage -geometry 500x100 "Could not start kdeinit5. Check your installation."
+  message "Could not start kdeinit5. Check your installation."
   exit 1
 fi
 
-qdbus org.kde.KSplash /KSplash org.kde.KSplash.setStage kinit &
+qdbus=qdbus-qt5
+$qdbus org.kde.KSplash /KSplash org.kde.KSplash.setStage kinit &
 
 # finally, give the session control to the session manager
 # see kdebase/ksmserver for the description of the rest of the startup sequence
@@ -324,13 +281,13 @@ if test x"$wait_drkonqi"x = x"true"x ; t
     # wait for remaining drkonqi instances with timeout (in seconds)
     wait_drkonqi_timeout=`kreadconfig5 --file startkderc --group WaitForDrKonqi --key Timeout --default 900`
     wait_drkonqi_counter=0
-    while qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
+    while $qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
         sleep 5
         wait_drkonqi_counter=$((wait_drkonqi_counter+5))
         if test "$wait_drkonqi_counter" -ge "$wait_drkonqi_timeout" ; then
             # ask remaining drkonqis to die in a graceful way
-            qdbus | grep 'org.kde.drkonqi-' | while read address ; do
-                qdbus "$address" "/MainApplication" "quit"
+            $qdbus | grep 'org.kde.drkonqi-' | while read address ; do
+                $qdbus "$address" "/MainApplication" "quit"
             done
             break
         fi
diff -up plasma-workspace-5.11.4/startkde/startplasma.cmake.startkde plasma-workspace-5.11.4/startkde/startplasma.cmake
--- plasma-workspace-5.11.4/startkde/startplasma.cmake.startkde	2017-11-28 06:44:10.000000000 -0600
+++ plasma-workspace-5.11.4/startkde/startplasma.cmake	2017-12-19 10:38:47.591485023 -0600
@@ -128,7 +128,8 @@ if test $? -ne 0; then
   exit 1
 fi
 
-qdbus org.kde.KSplash /KSplash org.kde.KSplash.setStage kinit
+qdbus=qdbus-qt5
+$qdbus org.kde.KSplash /KSplash org.kde.KSplash.setStage kinit
 
 # finally, give the session control to the session manager
 # see kdebase/ksmserver for the description of the rest of the startup sequence
@@ -157,13 +158,13 @@ if test x"$wait_drkonqi"x = x"true"x ; t
     # wait for remaining drkonqi instances with timeout (in seconds)
     wait_drkonqi_timeout=`kreadconfig5 --file startkderc --group WaitForDrKonqi --key Timeout --default 900`
     wait_drkonqi_counter=0
-    while qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
+    while $qdbus | grep "^[^w]*org.kde.drkonqi" > /dev/null ; do
         sleep 5
         wait_drkonqi_counter=$((wait_drkonqi_counter+5))
         if test "$wait_drkonqi_counter" -ge "$wait_drkonqi_timeout" ; then
             # ask remaining drkonqis to die in a graceful way
-            qdbus | grep 'org.kde.drkonqi-' | while read address ; do
-                qdbus "$address" "/MainApplication" "quit"
+            $qdbus | grep 'org.kde.drkonqi-' | while read address ; do
+                $qdbus "$address" "/MainApplication" "quit"
             done
             break
         fi
diff -up plasma-workspace-5.11.4/startkde/startplasmacompositor.cmake.startkde plasma-workspace-5.11.4/startkde/startplasmacompositor.cmake
--- plasma-workspace-5.11.4/startkde/startplasmacompositor.cmake.startkde	2017-11-28 06:44:10.000000000 -0600
+++ plasma-workspace-5.11.4/startkde/startplasmacompositor.cmake	2017-12-19 10:38:14.629269525 -0600
@@ -13,7 +13,7 @@ if [ -n "$bindir" ]; then
     *) PATH=$bindir:$PATH; export PATH;;
   esac
 else
-  qdbus=qdbus
+  qdbus=qdbus-qt5
 fi
 
 # We need to create config folder so we can write startupconfigkeys
